src/components/HathorTextInput.js

```diff
@@ -7,12 +7,17 @@
 
 import React from 'react';
 import { TextInput } from 'react-native';
+import { COLORS, STYLE } from '../styles/themes';
 
 const HathorTextInput = (props) => (
   <TextInput
     {...props}
     style={[{
-      width: 100, padding: 8, borderRadius: 4, borderColor: 'gainsboro', borderWidth: 1,
+      width: 100,
+      padding: 8,
+      borderRadius: 4,
+      borderColor: COLORS.borderColor,
```

Comment:
Change from `#DCDCDC` to `#eeeeee`

Reply:
Reverted to original color on [2e29c07](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/327/commits/2e29c070354d1fae1ee605c2884a2c6ff80cb0b2)